permissions/plugin-pipeline-cps-oras.yml

```diff
@@ -6,4 +6,6 @@ paths:
 developers:
 - "jonesbusy"
 issues:
-  - jira: 29725
+  - github: *GH
```

Comment:
But GH issues are disabled on the repo, so currently there is nowhere to report issues for this plugin at all.